src/app/styles/index.css

```
@@ -247,3 +247,58 @@ button:active:not(:disabled),
 }
+
+/* ─── Accessibility utilities ─────────────── */
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)